resources/python/ci/pytest.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -eu
+
+py3clean .
+rm -rf ./.cache
+rm -rf ./.pytest_cache
+rm -rf ./.mypy_cache
+rm -rf ./cov_report
+USE_DUMMY_DBUS=true \
+    PYTHONPATH=:../dbus-interface-lib:../libpalantir:../libPalantir:../charon:../libCharon:../libsmeagol:../libSmeagol:../marvin-service/src \
```

Comment:
These paths don't exist in my development environment. Maybe you'd have to add these as required repositories in the documentation?

Then again, I also don't really want my tests to fail if there is a mistake in libPalantir or something.